src/Sylius/Bundle/ApiBundle/Serializer/ProductImageNormalizer.php

```diff
@@ -80,7 +80,9 @@ private function resolvePath($data): array
             /** @var string $filter */
             $filter = $request->query->get('filter');
```

Comment:
```suggestion
            $filter = $request->query->getString('filter');
```
with removal of `/** @var string $filter */`